app/uk/gov/hmrc/agentassurance/controllers/PropertiesController.scala

```diff
@@ -16,57 +16,32 @@
 
 package uk.gov.hmrc.agentassurance.controllers
 
-
-import play.api.libs.json.Json
 import play.api.mvc.{Request, Result}
 import uk.gov.hmrc.agentassurance.repositories.PropertiesRepository
 import uk.gov.hmrc.play.bootstrap.controller.BaseController
 import uk.gov.hmrc.agentassurance.model._
-
 import scala.concurrent.ExecutionContext.Implicits.global
 import scala.concurrent.Future
 
-abstract class PropertiesController (repository: PropertiesRepository) extends BaseController {
+abstract class PropertiesController(repository: PropertiesRepository) extends BaseController {
 
   def key: String
 
+  protected def getCollectionUtrs(key: String)(implicit request: Request[Any]): Future[Result] = {
+    repository.getUtrsForKey(key).map(response => if(response.nonEmpty) Ok(response.mkString(",")) else NoContent)
```

Comment:
It should return an array instead of a string of utrs

Reply:
that is how it previously worked.